docs/setup/betamocks.md

```diff
@@ -58,16 +58,6 @@ Each service description has a `base_uri` (pulled from Settings)
 - `method:` a symbol of the http verb `:get`, `:post`, `:put`...
 - `path:` the path that combined with the base_uri makes a full URI
 - `file_path:` where to save the file (relative to betamocks' cache dir)
-```yaml
-:services:
-
-# EVSS::PCIUAddress
```

Comment:
You might need to replace this with Contact Information so other devs can use betamocks

Reply:
Sounds good, I'll make this update.

Reply:
How's this with an updated example?

Reply:
Hmm... I guess you can delete it. It's only for demographics.

Reply:
Sorry for the back and forth. Just remove it

Reply:
Updated!